src/xcode/ENA/ENA/Source/Workers/Store/SecureStore.swift

```diff
@@ -45,6 +45,7 @@ final class SecureStore: Store, AntigenTestProfileStoring {
 	func wipeAll(key: String?) {
 		do {
 			try kvStore.wipeAll(key: key)
+			antigenTestProfileSubject.send(nil)
```

Comment:
I think this is the real fix. The changes in the `ExpsoureSubmissionCoordinator` are not necessary and confuses me more then it helps. Do we really need the changes in the `ExpsoureSubmissionCoordinator`?

Reply:
The changes there are not necessary but I found that the store attribute is passed into that class unnecessarily. I just took it out to mitigate the confusion.

Reply:
Reverted